src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -105,10 +105,33 @@ def metadata_call_credentials(metadata_plugin, name=None):
     A CallCredentials object for use in a GRPCCallOptions object.
   """
   if name is None:
-    name = metadata_plugin.__name__
+    name = metadata_plugin.__class__.__name__
   return CallCredentials(
       _low.call_credentials_metadata_plugin(metadata_plugin, name))
 
+def google_call_credentials(credentials):
+  """Construct CallCredentials from google credentials
+
+  Args:
+    credentials: A google credentials object from oauth2client
```

Comment:
Don't be ambiguous about the type of this parameter - use the correct, formal name of the interface or class.